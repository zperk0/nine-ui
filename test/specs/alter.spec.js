import NAlter from '../../packages/n-alert';
import { mount } from '@vue/test-utils';

describe('NAlter', () => {
  it('renders correctly', () => {
    const wrapper = mount({
      render () {
        return <NAlter show={true}/>;
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
  it('renders correctly type', () => {
    const wrapper = mount({
      render () {
        return <NAlter show={true} type="info"/>;
      }
    });
    expect(wrapper.classes()).toContain('info');
  });
  it('renders correctly message', () => {
    const wrapper = mount({
      render () {
        return <NAlter show={true} message={'test'}/>;
      }
    });
    expect(wrapper.text()).toBe('test');
  });
});
