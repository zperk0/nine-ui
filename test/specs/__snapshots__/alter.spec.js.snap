// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NAlter renders correctly 1`] = `
<div class="base-alert">
  <div class="alert-content">
    <!----><span class="message-text"><!----></span>
    <!---->
  </div>
</div>
`;
